core/src/commonMain/kotlin/com/xebia/functional/xef/vectorstores/Memory.kt

```diff
@@ -2,4 +2,11 @@ package com.xebia.functional.xef.vectorstores
 
 import com.xebia.functional.xef.llm.models.chat.Message
 
+/**
+ * Representation of the memory of a message in a conversation.
+ *
+ * @property content message sent.
+ * @property conversationId uniquely identifies the conversation in which the message took place.
+ * @property timestamp in milliseconds.
```

Comment:
This is the important part, but I guess it's not that important now. The issue was that we are modelling `Memories` to have a `timestamp` of type `Long`. That value may be stored in a postgres database, and the schema required that field to be of type `TIMESTAMP`, so I needed to make sure how to transform from a `Long` to a `TIMESTAMP`. Right now that is avoided, because I changed the schema to use `BIGINT` instead of `TIMESTAMP`. More about that below